src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -92,19 +92,27 @@ private TupleTypeSymbol(ImmutableArray<Location> locations, NamedTypeSymbol unde
             return Create(locationOpt, underlyingType, elementLocations, elementNames);
         }
 
-        public static TupleTypeSymbol Create(NamedTypeSymbol tupleCompatibleType)
-        {
-            return Create(ImmutableArray<Location>.Empty,
-                          tupleCompatibleType,
-                          default(ImmutableArray<Location>),
-                          default(ImmutableArray<string>));
-        }
-
-        public static TupleTypeSymbol Create(Location locationOpt, NamedTypeSymbol tupleCompatibleType, ImmutableArray<Location> elementLocations, ImmutableArray<string> elementNames)
-        {
-            return Create(locationOpt == null ? ImmutableArray<Location>.Empty : ImmutableArray.Create(locationOpt),
-                          tupleCompatibleType, elementLocations, elementNames);
-        }
+        public static TupleTypeSymbol Create(NamedTypeSymbol tupleCompatibleType) =>
+            Create(ImmutableArray<Location>.Empty,
+                   tupleCompatibleType,
+                   default(ImmutableArray<Location>),
+                   default(ImmutableArray<string>));
+
+        public static TupleTypeSymbol Create(NamedTypeSymbol tupleCompatibleType,
+                                             ImmutableArray<string> elementNames) =>
+            Create(ImmutableArray<Location>.Empty,
+                   tupleCompatibleType,
+                   default(ImmutableArray<Location>),
+                   elementNames);
+
+        public static TupleTypeSymbol Create(Location locationOpt,
+                                             NamedTypeSymbol tupleCompatibleType,
```

Comment:
I'm not sure I understand -- this overload already existed, I just changed it to an expr-bodied method.

---
In reply to: [69643059](https://github.com/dotnet/roslyn/pull/12344#discussion_r69643059) [](ancestors = 69643059)